src/interface/interface.ts

```diff
@@ -161,13 +163,13 @@ export interface VerifyData {
 }
 
 export interface QueryParams {
-  user_address: string;
-  md5: string;
-  id: string;
-  tasktype: string;
-  taskstatus: string;
-  start?: number;
-  total?: number;
+  user_address: string | null;
+  md5: string | null;
+  id: string | null;
+  tasktype: TaskType | null;
+  taskstatus: TaskStatus | null;
```

Comment:
As we do not want to change Client's behavior, maybe leave it to string |  null.

Add/export TaskType/TaskStatus is correct code so we can still add TaskType, just need keep the QueryParams  interface to allow empty string